procedures/unit-testing-basics.ipf

```diff
@@ -2389,16 +2394,22 @@ End
 ///                         The Igor debugger will be left in its current state when running the
 ///                         tests.
 ///
+/// @param	debugMode      (optional) default disabled, enabled when set to 1-7: @n
```

Comment:
default disabled means 0, while for the user only 1 - 7 is allowed. It seems to be a bit strange that the default value is not part of the range that the user can set.
It does not hurt to include 0 for DEBUG_DISABLE as it is logical. There should also be something documented (briefly) for the case one uses allowDebug and debugMode at the same time.